hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -163,6 +163,19 @@
   private final RegionStates regionStates = new RegionStates();
   private final RegionStateStore regionStateStore;
 
+  /**
+   * When the operator uses this configuration option, any version between
+   * the current version and the new value of
+   * "hbase.min.version.move.system.tables" does not trigger any region movement.
+   * It is assumed that the configured range of versions do not require special
+   * handling of moving system table regions to higher versioned RegionServer.
```

Comment:
s/do/does/ and s/of//.  Perhaps point at the location in the code where this is done.

Can we add examples of how this would work -- the ones from the JIRA?

Reply:
Done, Thanks